pandas/tests/indexing/multiindex/test_chaining_and_caching.py

```diff
@@ -60,7 +60,9 @@ def test_cache_updating(using_copy_on_write):
             df.loc[0]["z"].iloc[0] = 1.0
         assert df.loc[(0, 0), "z"] == df_original.loc[0, "z"]
     else:
-        df.loc[0]["z"].iloc[0] = 1.0
+        # TODO(CoW-warn) should raise custom warning message about chaining?
+        with tm.assert_cow_warning(warn_copy_on_write):
```

Comment:
We could just enable the ChainedAssignmentError? Would probably have to adjust the message

Reply:
> We could just enable the ChainedAssignmentError? Would probably have to adjust the message

That's a good point. I am tackling warnings for chained setitem in https://github.com/pandas-dev/pandas/pull/55522, and there I was now using a "FutureWarning: ChainedAssignmentError ..." (so using it as the first word of the message, to make it stand out). I could also use the actual warning class, _if_ we are fine that it  then is no FutureWarning (given this is not a "typical" deprecation, but one to explicitly opt in to, that's probably fine).